.github/workflows/push.yml

```diff
@@ -103,7 +103,8 @@ jobs:
         uses: astral-sh/ruff-action@f14634c415d3e63ffd4d550a22f037df4c734a60 # v3.1.0
         with:
           version: "0.9.1"
-          args: "format --check"
+          # tagging.py is synced from universe in the `openapi/tagging` directory and follows different format rules.
+          args: "format --check --exclude tagging.py"
```

Comment:
Can you try this approach for excluding? https://github.com/databricks/cli/blob/main/acceptance/cmd/patchwhl/ruff.toml

you can add it to root ruff.toml.

Reply:
Thanks, works now